scss/jeet/_functions.scss

```diff
@@ -1,23 +1,20 @@
-/**
- * List functions courtesy of the wonderful folks at Team Sass.
- * Check out their awesome grid: Singularity.
- */
-
-/**
- * Get  percentage from a given ratio.
- * @param {number} [$ratio=1] - The column ratio of the element.
- * @returns {number} - The percentage value.
- */
+/// List functions courtesy of the wonderful folks at Team Sass.
```

Comment:
Triple slashes are intended to comment Sass items (variables, functions, mixins and placeholders). Everything else should be commented with either double slashes or C-style comments.